client/src/pages/index.tsx

```diff
@@ -54,151 +68,127 @@ const IndexPage = ({location}: IndexPageProps) => {
   return (
     <Layout location={location}>
 
-      <J40MainGridContainer fullWidth={true}>
```

Comment:
Has design removed the Alert on this page? Sprint 4 designs show them to exist. Not sure tho...

Reply:
Oh, good catch. Re-added for now.

Reply:
I'll start working on fixing alerts as soon as the PR is in.